lib/git_reflow/git_helpers.rb

```diff
@@ -40,11 +40,12 @@ def fetch_destination(destination_branch)
       run_command_with_label "git fetch origin #{destination_branch}"
     end
 
-    def update_destination(destination_branch)
+    def update_destination(destination_branch, remote = 'origin')
```

Comment:
With the rebase in here, we're doing more than updating just the destination. This not only makes the naming no longer as clear on intent, but also adds some unintended side affects to the `deliver` as it will now rebase master.

I do like the addition of the `remote` option though.  Let's add that in as a hash though so we don't have to guess as to what that value represents when we call it.  So `def update_destination(destination_branch, options = {})`.